packages/devtools_app/lib/src/flutter/flutter_widgets/linked_scroll_controller.dart

```diff
@@ -0,0 +1,369 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+// This file is copied from package:flutter_widgets.
```

Comment:
Will this remain relevant in a year? Should this detail be kept in the commit message exclusively?

Reply:
added some context to the comments to state that they were "originally" copied from flutter_widgets and that the source may diverge over time.